rotations.yml

```diff
@@ -104,19 +105,18 @@ pools:
       - "Orion"
       - "Oumuamua"
       - "Pandemic"
-      - "Persisto"
+      - "Perspookysto"
       - "Pineium DTM"
-      - "Pipes and Punters"
```

Comment:
I think you accidentally switched this out for a different Halloween map.

Reply:
Is the Halloween version not Boo Pipes?